_data/community_tracers_instrumentation.yml

```
@@ -30,13 +30,21 @@
 - language: Java
   library: >-
     [Spring Cloud Sleuth](https://github.com/spring-cloud/spring-cloud-sleuth)
-    [Micrometer Tracing](https://github.com/micrometer-metrics/tracing)
   framework: Spring, Spring Cloud (e.g. Stream, Netflix)
   propagation: Http (B3), Messaging (B3)
   transports: Http, Spring Cloud Stream Compatible (e.g. RabbitMQ, Kafka, Redis or anything with a custom Binder)
   sampling: "Yes"
   notes: Java 7 or higher
 
+- language: Java
+  library: >-
+    [Micrometer Tracing](https://github.com/micrometer-metrics/tracing)
+  framework: Spring, Quarkus, Micronaut, Helidon
```

May seem like a nit, but I think this question has come up (what about Spring Boot 2)

```suggestion
  framework: Spring Boot 3+, Quarkus, Micronaut, Helidon
```

---

technically, Spring Framework 6+

I'm not sure if Quarkus, Micronaut and Helidon support Micrometer Tracing besides Metrics.

For example, in the Helidon doc, only OpenTelemetry is referred.
https://helidon.io/docs/v4/mp/telemetry

---

Updated to "Spring Framework 6+". About frameworks related ...?

---

> Quarkus, Micronaut, Helidon
Have you tested that it works?
If not, it's probably safe to remove the mention.

---

> > Quarkus, Micronaut, Helidon
> 
> Have you tested that it works? If not, it's probably safe to remove the mention.

I have NOT had time to tested it. Should I remove it for safety reasons?

---

I would remove them. Iooked at the other docs and unlike spring boot, there are no zipkin tracing integrations/registries/instructions in Quarkus, Micronaut, Helidon. The closest they come is otel, and that's stretch to say is a community integration for zipkin.

p.s. @making still think it should be about spring boot because spring itself has no zipkin integration. It is a bit misleading and inconsistent to say Spring Framework 6+ which doesn't have an integration path for zipkin, unless I'm missing something https://docs.spring.io/spring-boot/reference/actuator/tracing.html

---

updated @codefromthecrypt